### ISSUE TITLE
Run Executor

### ISSUE THREAD

**Issue by jackzampolin** [gelatodigital/gelato-network#268]
I'm not seeing in the docs how I can run an Executor. Am I missing this somewhere

**Comment by hilmarx** [comment]
Yes the docs are still incomplete atm in that regard as we are currently focussed on working on the v2 that will introduce a lot of changes on the Executor side, therefore we don't want community members to invest too many resources in the current version of the Executor system: 

If you still want to try it out: Running an Executor basically requires you to fetch the TaskReceipts assigned to your Executor via `LogTaskSubmitted` on GelatoCore.sol. For the `taskReceipts` that got assigned to your executors, call `canExec` to see if it's executable and if so, call `exec` to execute the task and get the reward (gasConsumed * gasPrice + fee). There are of course some nuances to that. However, I would recommend waiting for the next version which will come with better documentation and is estimated to arrive beginning of next year!
